lib/pages/onboarding_page/widgets/add_budget.dart

```diff
@@ -0,0 +1,88 @@
+import 'package:flutter/material.dart';
+import 'package:flutter_riverpod/flutter_riverpod.dart';
+import 'package:sossoldi/constants/functions.dart';
```

Comment:
You use a mix of relative and absolute path import, we try to use relative import across the project, just an easy fix, i noticed that also in routes.dart and some other widgets :)